app/src/main/res/values/strings.xml

```diff
@@ -18,49 +18,49 @@
-    <string name="notification_channel_sync_warnings_desc">Non-fatal synchronization problems like dismissed changes of read-only collections</string>
+    <string name="notification_channel_sync_warnings_desc">Small syncing problems like dismissed changes of read-only collections</string>
     <string name="notification_channel_sync_status">Status messages</string>
     <string name="notification_channel_sync_status_desc">Informational status messages like the sync change summary</string>
 
     <!-- Crash -->
-    <string name="crash_title">EteSync has crashed!</string>
+    <string name="crash_title">EteSync crashed!</string>
     <string name="crash_message">Please send stack trace to developers.</string>
     <string name="crash_email_subject">EteSync Debug Info</string>
-    <string name="crash_email_body">If possible, please include any other relevant information such as what you did to trigger this error.</string>
+    <string name="crash_email_body">If possible, please include any other relevant info, such as what you did before this happened.</string>
 
     <!-- tourguide -->
     <string name="tourguide_title">Did you know?</string>
 
     <!-- startup dialogs -->
     <string name="startup_battery_optimization">Battery Optimization</string>
-    <string name="startup_battery_optimization_message">Android may disable/reduce EteSync synchronization after a few days. To prevent this, turn off battery optimization.</string>
+    <string name="startup_battery_optimization_message">Turn off battery optimization to prevent Android turning off/reducing EteSync syncing after a few days.</string>
     <string name="startup_battery_optimization_disable">Turn off for EteSync</string>
     <string name="startup_dont_show_again">Don\'t show again</string>
     <string name="startup_development_version">EteSync Preview Release</string>
-    <string name="startup_development_version_message">This is a development version of EteSync. Be aware that things may not work as expected. Please give us constructive feedback to improve EteSync.</string>
-    <string name="startup_development_version_give_feedback">Give feedback</string>
+    <string name="startup_development_version_message">This is a development version of EteSync. Be aware that things may not work as expected. Please give us constructive feedback to improve it.</string>
+    <string name="startup_development_version_give_feedback">Provide feedback</string>
     <string name="startup_vendor_specific_bugs">Potential Vendor Bugs</string>
-    <string name="startup_vendor_specific_bugs_message">EteSync has detected you are using an Android version that may contain vendor specific bugs.\nPlease take a look at the FAQ for more information.</string>
+    <string name="startup_vendor_specific_bugs_message">You are using an Android version that may contain vendor specific bugs.\nPlease take a look at the FAQ for more info.</string>
```

Comment:
This feels less clear to me. The problem is that after the change of string it could be read as if the EteSync app's Android version is buggy, rather the Android system itself. That's why we had the verbose wording, to be clear it's not an issue with the EteSync app but rather with their Android version.

Reply:
The user still has to know what an "Android vendor" is, which is not going to happen. If anything the error could be in the EteSync detection with the earlier string.
How about something like "Upgrade to something newer than Android 6.0 to avoid bugs in it that cause ____"

Reply:
It's not an Android 6.0 thing. It's vendor bugs, that is, bugs that are introduced by the vendors like Samsung, Huawei and etc. They could manifest in all Android versions, so upgrading won't help.
Also, in my point of view, "information" is better than "info" in this context.

How about:
"You are using an Android device that may contain vendor specific bugs.\nPlease take a look at the FAQ for more information."
?

Reply:
Implemented. Used the latter sentence from elsewhere for consistency.